tasks/install-cordova.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Copy npm script
+  template: src=setup_npm_cordova.sh dest=/Users/{{ work_user }}/{{ work_dir }} mode=0777
+
+- name: Run npm script
+  command: sh /Users/{{ work_user }}/{{ work_dir }}/setup_npm_cordova.sh
+  tags: monit,skip_ansible_lint
```

Comment:
Why have you declared these tags?

Reply:
when running lint on a playbook, it will not swear at the "command" module used